libs/portal-framework-ui/src/components/data-table/Toolbar.tsx

```
@@ -0,0 +1,148 @@
+
+  // Access the rest of the properties from refineTable
+  const { setFilters, setSorters, tableQuery, filters, sorters } =
+    refineTable || {};
+
+  // If no toolbar config is provided through contexts, don't render anything
+  if (!config) {
+    return null;
+  }
+
+  // Sort items by order if specified
+  const sortedItems = React.useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)